ctrd/container.go

```diff
@@ -221,28 +221,28 @@ func (c *Client) DestroyContainer(ctx context.Context, id string) (*Message, err
 }
 
 // CreateContainer create container and start process.
-func (c *Client) CreateContainer(ctx context.Context, container *Container) error {
+func (c *Client) CreateContainer(ctx context.Context, container *Container) (uint32, error) {
```

Comment:
This change will break containerd client's API. Please add more details instructions for the change, and share the feedback from @skoo87 . Thanks a lot, @CodeJuan

Reply:
Yeah. It's better to get PID by using the interface of ctrd.